Modules/System/AL Rest Client/src/ALHttpContent.Codeunit.al

```diff
@@ -0,0 +1,261 @@
+/// <summary>Holder object for the Http Content data.</summary>
+codeunit 2354 "AL Http Content"
```

Comment:
No need to prefix any of the objects with `AL`. The consumers could easily differentiate between the built-in HttpClient and this one by type.

Reply:
Removed the prefix "AL" everywhere.